capture.go

```diff
@@ -16,10 +16,14 @@ type CaptureClient struct {
 type Capture struct {
 	CaptureParams
 
-	ID           string       `json:"id"`
-	Result       Result       `json:"result"`
-	Created      json.Number  `json:"created"`
-	ProviderData ProviderData `json:"provider_data"`
+	ID                    string                `json:"id"`
+	Result                Result                `json:"result"`
+	Created               json.Number           `json:"created"`
+	ProviderData          ProviderData          `json:"provider_data"`
+	ProviderSpecificData  DecodedJSON           `json:"provider_specific_data"`
+	Level23               Level23               `json:"level_2_3,omitempty"`
+	ProviderConfiguration ProviderConfiguration `json:"provider_configuration,omitempty"`
```

Comment:
Agree it looks strange but some fields aren't required in the documentation and don't have the tag `omitempty`  other fields have this tag but still - aren't required. I can't explain the difference here and it's the reason because I have added it somewhere.

Reply:
Removed omitempty in the structs for responses.